admin/filters/class-abstract-post-filter.php

```diff
@@ -49,6 +49,26 @@ public function register_hooks() {
 		}
 
 		add_filter( 'posts_where', array( $this, 'filter_posts' ) );
+
+		if ( $this->is_filter_active() && $this->get_explanation() !== null ) {
```

Comment:
You might consider swapping the checks. I think `$this->get_explanation` might be cheaper to do first.